docs/advanced-features/custom-error-page.md

```diff
@@ -2,9 +2,34 @@
 description: Override and extend the built-in Error page to handle custom errors.
 ---
 
-# Custom Error Page
+## 404 Page
 
-**404** or **500** errors are handled both client-side and server-side by the `Error` component. If you wish to override it, define the file `pages/_error.js` and add the following code:
+A 404 page may be accessed very often. Rendering the `_error` page for every visit increases the load for your server or the invocations for serverless functions. This can result in increased costs and slow experiences.
```

Comment:
Because we haven't talked about `_error` yet, maybe we should do this:

```suggestion
A 404 page may be accessed very often. Server-rendering an error page for every visit increases the load for your server or the invocations for serverless functions. This can result in increased costs and slow experiences.
```